src/function/matrix/eigs/complexEigs.js

```diff
@@ -444,30 +439,19 @@ export function createComplexEigs ({ addScalar, subtract, flatten, multiply, mul
       solutions.shift() // ignore the null vector
 
       // looks like we missed something, try inverse iteration
+      // But if that fails, just presume that the original matrix truly
+      // was defective.
       while (solutions.length < multiplicities[i]) {
```

Comment:
Question: I tried to see where the "actual" fix is for dealing with defective matrices but I'm not sure. Is it the removal of the logic around `failedLambdas` here?

Reply:
Three ingredients were necessary for the "actual" numerical fix: (1) accepting that if inverse iteration completely fails it's presumably because the matrix is actually defective (not a guarantee, but likely correct); (2) fixing the computation of the orthogonal component below; and (3) fixing the computation of the "jacobian2x2" which had a significant error in it.

Reply:
O wow, thanks for the clarification.